src/bananas.js

```diff
@@ -0,0 +1,180 @@
+define([
+    'vellum/xpath',
+], function(
+    xpath
+) {
+    var OUTSIDE_BANANA = 0,
+        INSIDE_BANANA = 1,
+        DELIMITER = "🍌",
```

Comment:
lol. i figured it would be the least likely candidate to already be used in a file